recipes/libgeoip/build.bat

```diff
@@ -0,0 +1,5 @@
+GEOIPDATADIR=$PREFIX
```

Comment:
Use `%LIBRARY_PREFIX%` instead of `$PREFIX`
Also you need to set `INSTDIR` to the same path.